tools/ci/docker/linux/Dockerfile

```diff
@@ -191,6 +191,12 @@ RUN cd renesas-tools/source/newlib && \
 RUN cd /tools/renesas-tools/build/gcc && \
   make && make install
 
+# Install Rust and targets supported from NuttX
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
```

Comment:
What is a destination folder for `curl`?

Reply:
It's piped to "sh" and than directly executed. That's there official way from Rust itself to install the suite. And that seems to work well (log taken from 5567):

    #51 [nuttx-toolchain-renesas 9/9] RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs/ | sh -s -- -y &&   $HOME/.cargo/bin/rustup target add thumbv6m-none-eabi &&   $HOME/.cargo/bin/rustup target add thumbv7m-none-eabi
    #51 sha256:9445c613b057da3a0a56958aa76f14669e41ecf46516155f09eeab6967d8a7d2
    #51 0.499 info: downloading installer
    #51 0.800 info: profile set to 'default'
    #51 0.800 info: default host triple is x86_64-unknown-linux-gnu
    #51 0.801 info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
    #51 0.993 info: latest update on 2022-01-20, rust version 1.58.1 (db9d1b20b 2022-01-20)
    #51 0.994 info: downloading component 'cargo'
    #51 1.090 info: downloading component 'clippy'
    #51 1.125 info: downloading component 'rust-docs'
    #51 1.349 info: downloading component 'rust-std'
    #51 1.643 info: downloading component 'rustc'
    #51 2.253 info: downloading component 'rustfmt'
    #51 2.309 info: installing component 'cargo'
    #51 3.093 info: installing component 'clippy'
    #51 3.478 info: installing component 'rust-docs'
    #51 7.031 info: installing component 'rust-std'
    #51 10.05 info: installing component 'rustc'
    #51 15.65 info: installing component 'rustfmt'
    #51 16.21 info: default toolchain set to 'stable-x86_64-unknown-linux-gnu'
    #51 16.21 
    #51 16.23   stable-x86_64-unknown-linux-gnu installed - rustc 1.58.1 (db9d1b20b 2022-01-20)
    #51 16.23 
    #51 16.23 
    #51 16.23 Rust is installed now. Great!
    #51 16.23 
    #51 16.23 To get started you may need to restart your current shell.
    #51 16.23 This would reload your PATH environment variable to include
    #51 16.23 Cargo's bin directory ($HOME/.cargo/bin).
    #51 16.23 
    #51 16.23 To configure your current shell, run:
    #51 16.23 source $HOME/.cargo/env
    #51 16.45 info: downloading component 'rust-std' for 'thumbv6m-none-eabi'
    #51 16.82 info: installing component 'rust-std' for 'thumbv6m-none-eabi'
    #51 17.98 info: downloading component 'rust-std' for 'thumbv7m-none-eabi'
    #51 18.12 info: installing component 'rust-std' for 'thumbv7m-none-eabi'
    #51 DONE 21.8s

It's working, because the targets could be installed.